core/core_constants.cpp

```diff
@@ -508,6 +508,7 @@ void register_global_constants() {
 
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_HINT_NONE);
 	BIND_CORE_ENUM_CONSTANT(PROPERTY_HINT_RANGE);
+	BIND_CORE_ENUM_CONSTANT(PROPERTY_HINT_ANGLE);
```

Comment:
This should move down after KEY_ACCEL to match the enum order, and have docs properly ordered by value.

Reply:
let me do some further changes about this, I am not kind of happy with this hint.